.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,42 @@
-Resolves #
-
-Make sure you've covered the following bases in your PR. They won't all be
-needed for every PR (ex. fixing a straightforward rubocop issue might have no
-tradeoffs and add no dependencies).
-
-- [ ] Did you fill in the `Resolves #<issue number>` in the first line?
-- [ ] Is this a work in progress? If so, put "WIP" in the title.
-- [ ] What were the main changes/goals of this PR?
-- [ ] Did you need to make any tradeoffs? Why did you make the choice you made?
-- [ ] Did you add any dependencies?
-- [ ] Anything else we should know about?
+<!--Read comments, before commiting pull request read checklist again
+
+# Checklist:
+
+- I have performed a self-review of my own code
+- I have commented my code, particularly in hard-to-understand areas
+- I have made corresponding changes to the documentation
+- I have added tests that prove my fix is effective or that my feature works
+- New and existing unit tests pass locally with my changes
+-->
+
+# Resolves #001 <!--fill issue number-->
+
+# Description
+<!-- Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
+List any dependencies that are required for this change. Put "WIP" in the title if work is in progress. Did you add any dependencies?
+Include anything else we should know about. -->
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+- [ ] Documentation update
+
+# How Has This Been Tested?
+<!--Optional. Delete if not relevant-->
+
+<!-- Please describe the tests that you ran to verify your changes. 
+Provide instructions so we can reproduce. 
+Please also list any relevant details for your test configuration -->
```

Comment:
Since automated tests are required for feature changes, I'm hesitant to use the word "tests" (esp. with a checklist) here; people might just describe their specs.

Could you change this to something like "Do we need to do anything else to verify your changes? If so, provide instructions (including any relevant configuration) so that we can reproduce."?